### ISSUE TITLE
Predicting squiggles with taiyaki

### ISSUE THREAD

**Issue by nemitheasura** [nanoporetech/taiyaki#96]
Hello, 

I'm quite new to this field, so I would appreciate if you would like to guide me.

I'd like to use taiyaki to predict squiggles from given RNA sequences (in the end, I'd like to obtain a set of artificial squiggles mimicking the real ones). However, I can't find an instruction how to use predict_squiggle.py.  

Could you provide any tips?

**Comment by marcus1487** [comment]
`predict_squiggle.py` produces a prediction for the signal level, signal standard deviation and mean dwell for each reference base. This command does not directly produce squiggles (raw signal). We do not currently have a tool to produce "realistic" predicted squiggles. Depending on how realistic one might want these predicted raw signals to be, this can be quite a challenging task.

**Comment by tmassingham-ont** [comment]
The [scrappie](https://github.com/nanoporetech/scrappie) tool can predict RNA squiggles from sequence and this might be a better starting point for your work.

**Comment by nemitheasura** [comment]
Thank you for your response. 
I've tested scrappie beforehand and the output produced by it was not reliable in terms of what I wanted to simulate. DeepSimulator has better prediction models and is more reliable, but only in case of the DNAseq (it lacks pretrained RNA models). This is why I decided to give a try to taiyaki. I'm not that skilled to design my own prediction model and implement it to DeepSimulator - so I asked this question. Well, thank you anyways!

Have a nice day!
